scripts/scil_tractogram_dps_math.py

```diff
@@ -61,9 +61,15 @@ def _build_arg_parser():
                         '"delete" operations.')
 
     import_args = p.add_argument_group('Operation "import" mandatory options')
-    import_args.add_argument('--in_dps_file',
+    import_excl = import_args.add_mutually_exclusive_group()
+    import_excl.add_argument('--in_dps_file',
                              help='File containing the data to import to\n'
-                                  'streamlines (.txt, .npy or .mat).')
+                                  'streamlines (.txt, .npy or .mat). There\n'
+                                  'must be the same amount of entries as\n'
+                                  'there are streamlines.')
+    import_excl.add_argument('--in_dps_file_single_value',
```

Comment:
I think it would be better if instead of supplying a file we could suppy a value directly on the command line.

Reply:
I did it this way because dps is often an array. I don't think that would work in a command line. :)